src/eigen/preconditioners.jl

```diff
@@ -50,7 +63,7 @@ end
     if P.mean_kin === nothing
         ldiv!(Y, Diagonal(P.kin .+ P.default_shift), R)
     else
-    parallel_loop_over_range(1:size(Y, 2)) do n
```

Comment:
Removing parallelisation ok ?

Reply:
I need to double check performance, but this also goes in the direction of eventually branching out LOBPCG  from DFTK.

Reply:
I see, but I suppose the preconditioner would remain in DFTK in any case.